libs/shared/ui-components/src/Header/HeaderDesktop/HeaderDesktopNavigationProvider/HeaderDesktopDropdownLink.tsx

```diff
@@ -19,7 +19,7 @@ export const HeaderDesktopDropdownLink: FC<THeaderDesktopDropdownLinkProps> = ({
 
   return (
     <button
-      className="text-[1.7rem] font-normal leading-[2.225rem] text-left capitalize transition-colors motion-reduce:transition-none ease-in-out hover:text-green font-heading focus:text-green"
+      className="text-[1.7rem] font-normal leading-[2.225rem] text-left capitalize transition-colors motion-reduce:transition-none ease-in-out hover:text-darkgreen focus:text-darkgreen font-heading"
```

Comment:
Here I changed the text color from green to a color I aliased to "darkgreen" (558300) to make it AA compliant.

Reply:
I'm pretty sure that the green used for this hover state is a brand color, so we probably need to talk to the design team or come up with an alternative approach that still follows the brand guidelines. Maybe there's another green (or other color) in the brand that works better? Though since we're using it to work on both very dark and very light backgrounds in this case, I am not confident the same green will have enough contrast for both.

I have a few other ideas for long term! I know with the timeline they are unlikely to happen in this PR.
- Since there's only one menu that has that drop down, why not come up with another way to represent that information. Maybe the landing page for the section has all those options listed at the top?
- Don't rely on color (or solely on color) to signal the hover state. Use an icon or an underline or something similar.
- Change the background color of the menu to be dark, like the main navigation elsewhere.
- Work with the brand stewards to come up with an extension of the existing color palette that gives a greater range of values to work with in this situation. Maybe there is a dark equivalent of the brand green they'd like to incorporate.
- A combination of the above options.
- Experiment with CSS text shadow options. I do not recommend this approach when this website does not have other soft or low opacity treatments, but it is an option to try if nothing else works.